SConstruct

```diff
@@ -17,7 +17,13 @@ env = SConscript("godot-cpp/SConstruct")
 if env["platform"] == "windows":
     env.Append(CPPPATH=['include/'])
     env.Append(LIBPATH=['lib/SDL2/'])
-    env.Append(LIBS=['SDL2.dll.dll'])
+    print("CXX is: " + env["CXX"])
+    if env["CXX"] == "$CC":
```

Comment:
Does "$CC" represent the msvc compiler here? Just curious cause for mingw it was a lot longer, something like "mingw32_64_something"

Reply:
I just tested what the print-statement in line 20 printed and used it. It prints "CXX is: g++" when compiling with minGW in windows Command Prompt (command "scons use_mingw=true") and "CXX is: $CC" when using Visual C++ (command just "scons").

godot-cpp/SConstruct actually seems to determine the compiler another way (line 236+):
```
# Require C++17
if env.get("is_msvc", False):
    env.Append(CXXFLAGS=["/std:c++17"])
else:
    env.Append(CXXFLAGS=["-std=c++17"])
```

Tested this and it seems to work in the place of the previous check. I can commit this change if you like(?) "is_msvc" feels  more self-descriptive.